### PR TITLE
Flamegraph trimming bugfix, Disabling flamegraph trimming by default & fixed the compare legend visibility

### PR DESCRIPTION
**Pull request by manojVivek**

### CODE REVIEW

**Review by yomete — APPROVED** [review]

File: pkg/query/flamegraph_table.go [hunk 1]
```diff
@@ -543,15 +543,14 @@ func trimFlamegraphNodes(ctx context.Context, tracer trace.Tracer, nodes []*quer
 		if node.Cumulative < threshold {
 			continue
 		}
+		var oldChildren FlamegraphChildren = node.Children
```
Comment (asubiotto) [inline pkg/query/flamegraph_table.go:546]:
```suggestion
		oldChildren := node.Children
```

File: pkg/query/flamegraph_table.go [hunk 2]
```diff
@@ -543,15 +543,14 @@ func trimFlamegraphNodes(ctx context.Context, tracer trace.Tracer, nodes []*quer
...
+		flat := node.Cumulative - oldChildren.Cumulative()
 		var children FlamegraphChildren = trimFlamegraphNodes(ctx, tracer, node.Children, threshold)
```
Comment (asubiotto) [inline pkg/query/flamegraph_table.go:548]:
```suggestion
		children := trimFlamegraphNodes(ctx, tracer, node.Children, threshold)
```

File: pkg/query/flamegraph_table.go (hunk 1 above)
Comment (manojVivek) [reply pkg/query/flamegraph_table.go:546]: I was trying to do this but was unable to as `node.Children` ie `[]*querypb.FlamegraphNode` is not an interface. 

So, I created an alias type `FlamegraphChildren` and added some utility functions like below:

```
type FlamegraphChildren []*querypb.FlamegraphNode

func (n FlamegraphChildren) Cumulative() int64 {
	cumulative := int64(0)
	for _, child := range n {
		cumulative += child.Cumulative
	}
	return cumulative
}
```

I was unable to typecast and access these utility functions, so defined a variable with this type and used those for accessing the functions. 

There could a different way for this problem, please let me know.

Comment (asubiotto) [reply pkg/query/flamegraph_table.go:546]: ah, then what you have is fine, another option is:
```
oldChildren := FlamegraphChildren(node.Children)
```

Comment (manojVivek) [reply pkg/query/flamegraph_table.go:546]: Cool, thanks for the tip!

**Pull request merged**